test-cases/gutenberg/reusable.md

```diff
@@ -74,10 +75,13 @@ The reusable blocks page can be also used for editing and deleting blocks.
 **Deleted reusable block**
 
 **Steps**
-- Create two reusable blocks.
+- Navigate to the [reusable blocks page](https://wordpress.com/types/wp_block).
```

Comment:
Wdyt about a quick nudge to which platform this part of the steps should be performed on? Something like:
```suggestion
- On a web browser, navigate to the [reusable blocks page](https://wordpress.com/types/wp_block).
```

Reply:
Sure! I'll update this part in both tests `TC003` and `TC004`.

Reply:
I've just changed the steps that referenced the navigation to the reusable blocks page in this [commit](https://github.com/wordpress-mobile/test-cases/pull/79/commits/34b73333100a1c0a8c319d7860230361f5f1a85f).